src/ServiceControl.Plugin.CustomChecks.Sample/EndpointConfig.cs

```diff
@@ -1,19 +1,19 @@
 
 namespace ServiceControl.Plugin.CustomChecks.Sample
 {
-    using NServiceBus;
-    
+	using NServiceBus;
+	
 
-    /*
+	/*
 		This class configures this endpoint as a Server. More information about how to configure the NServiceBus host
 		can be found here: http://particular.net/articles/the-nservicebus-host
 	*/
-    public class EndpointConfig : IConfigureThisEndpoint, AsA_Server
```

Comment:
I still see value in acceptance tests, we can just fake SC and check that a message arrives and has the expected values. That would IMO catch most issues?